templates/trusted-node/cdk-node-config.toml

```
@@ -3,7 +3,7 @@ ForkUpgradeNewForkId = 0
 
 [Common]
 IsValidiumMode = {{.is_cdk_validium}}
-ContractVersions = "elderberry"
+ContractVersions = "banana"
```

I think we should still support `fork9` here.

```suggestion
{{if eq .zkevm_rollup_fork_id 9}}
ContractVersions = "elderberry"
{{else if eq .zkevm_rollup_fork_id 12}}
ContractVersions = "banana"
{{end}}
```

---

I think that in order to be retro compatible we would need to do more than just that, as `elderberry` is supposed to be run with the old stack (not using `cdk` node)

---

Yeah but we have options to specify this in `params.yml` :)

---

looks like this depens only on fork id after merging changes?